### DISCUSSION TITLE
Time Picker Set Interval

### DISCUSSION THREAD

**Discussion by toomanylogins** [coreui/coreui#701] [General]
Pro user. 
Is it possible to set an interval for dropdown ie 0,15,30,45 mins and also disable seconds ?
Note I tried your web form a couple of times but keep getting "Sorry, it seems that our mail server is not responding. "
Paul

**Comment by mrholek** [comment]
Hi,

At this moment it's not possible, but we have it on our roadmap for Q1, I can add a higher priority to deliver it faster.

BTW. I just checked our mail server and everything is ok.

**Comment by toomanylogins** [comment]
Thanks for reply Q1 is good for me. If I get the mail server message again will post here.
